lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -419,6 +419,10 @@ class TypeAnalyzer : public ForwardDataFlowAnalysis<ValueKnowledge> {
       return visitAtenSoftmaxLikeOp(logSoftmaxIntOp, operands);
     } else if (auto numToTensorOp = dyn_cast<PrimNumToTensorScalarOp>(op)) {
       return visitNumToTensorOp(numToTensorOp);
+    } else if (auto addCMulOp = dyn_cast<AtenAddCMulOp>(op)) {
```

Comment:
Can be replaced with if (isa<AtenAddCMulOp, addCDivOp>(op)) once we replace the template below with `Operation *op`